microsoft-edge/webview2/get-started/wpf.md

```diff
@@ -18,402 +18,428 @@ In this article, get started creating your first WebView2 app in Windows Present
-      The create a new project panel displays WPF app search results. The highlighted button is WPF app .NET core Windows Presentation Foundation client application for C#, Windows, and desktop. The dialog box displays a Next button.
-          :::image-end:::
+   The highlighted card in the following image is **WPF Application: .NET Core WPF Application**:
+    
+   ![The 'Create a new project' panel with the card selected, 'WPF Application: .NET Core WPF Application'.](media/wpf-getting-started-wpf-core.png)
 
-1.  For **WPF App (.NET Core)** enter values for **Project name** and **Location**, then click **Next**.
+   Alternatively, the highlighted card in the following image is **WPF App (.NET Framework): Windows Presentation Foundation client application**:
 
-    :::image type="complex" source="./media/wpf-getting-started-create-core.png" alt-text="Configure your new project WPF application dialog box displays project name, location, and solution name text boxes.":::
-      The configure your new project WPF application dialog box displays project name, location, and solution name text boxes. The dialog box displays Back and Next buttons.
-    :::image-end:::
+   ![The 'Create a new project' panel with the card selected, 'WPF App (.NET Framework): Windows Presentation Foundation client application'.](media/wpf-getting-started-wpf-fw.png)
 
-    1.  Choose **.NET Core 3.1** or later.
+   The **Configure your new project** WPF application dialog box appears.
 
-        :::image type="complex" source="./media/wpf-getting-started-create-core-add-info.png" alt-text="Additional information dialog box displays a drop-down menu for target framework.":::
-          The Additional information dialog box displays a drop-down menu for target framework. Choices are .NET core 3.0, 3.1. and 5.0. The dialog box displays Back and Create buttons.
-        :::image-end:::
+   ![The 'Configure your new project' WPF application dialog box.](media/wpf-getting-started-create-core.png)
 
-1. For **WPF App (.NET Framework)** enter values for **Project name** and **Location**, then choose **.NET Framework 4.6.2** or later.
- 
-    :::image type="complex" source="./media/wpf-getting-started-create-fw.png" alt-text="Configure your new project WPF app .NET framework dialog box displays project name, location, and solution name text boxes.":::
-      The configure your new project WPF app .NET framework dialog box displays project name, location, and solution name text boxes. The framework drop-down menu displays .NET framework versions including 4.6.1 and 4.7.2. The dialog box displays Back and Create buttons.
-    :::image-end:::
+1.  Enter values for **Project name** and **Location**, and then click **Next**.
+
+   The **Additional information** dialog box appears, with a **Target Framework** dropdown list.  Choices are **.NET Core 3.0**, **3.1**, and **5.0**:
```

Comment:
Lines 57 and 59, text and image about “Additional information” are not indented, breaking the numbered list.

Reply:
thanks for the fix